pallets/common/src/traits/asset.rs

```diff
@@ -21,7 +21,7 @@ use polymesh_primitives::{
 use polymesh_primitives_derive::VecU8StrongTyped;
 use sp_std::prelude::*;
 
-pub const GAS_LIMIT: u64 = 1_000_000_000;
+pub const GAS_LIMIT: u64 = 13_000_000_000;
```

Comment:
IMO this should be a parameter defined in the runtime.

Reply:
It doesn't add any value but we can do it for consistency.